test/cli_network_test.go

```diff
@@ -285,6 +285,27 @@ func (suite *PouchNetworkSuite) TestNetworkCreateWithOption(c *check.C) {
 	defer command.PouchRun("network", "remove", funcname)
 }
 
+// TestNetworkCreateWithIPAMOption creates network with ipam options
+func (suite *PouchNetworkSuite) TestNetworkCreateWithIPAMOption(c *check.C) {
+	pc, _, _, _ := runtime.Caller(0)
```

Comment:
Oh, we do not like this way to get the function name as container name.
Please just use `containerName := "TestNetworkCreateWithIPAMOption"`.

If somewhere also uses this wrong way, we need to change that.

Test code readability is the first thing we need I think. So we have the thought above.

 @xiaoxubeii

Reply:
maybe we should update the go-check so that we can use `c.TestName()` to get the name.

https://godoc.org/gopkg.in/check.v1#C.TestName